docs/source/using/image-caching.adoc

```diff
@@ -4,39 +4,123 @@ include::variables.adoc[]
 :icons:
 :toc: macro
 :toc-title:
-:toclevels: 1
+:toclevels: 2
 
 toc::[]
 
 [[image-caching-overview]]
 == Overview
 
-To speed up provisioning of the OpenShift cluster and to minimize network traffic, the core OpenShift images can be cached on the host.
-This feature is considered experimental and needs to be explicitly enabled.
+To speed up the provisioning of the OpenShift cluster and to minimize network traffic, container images can be cached on the host.
+These images can then be imported into the running Docker daemon, either explicitly on request or implicitly during xref:../using/basic-usage.adoc#minishift-start-overview[`minishift start`].
+The following sections describe image caching and its configuration in more detail.
 
-[[image-chaching-configuration]]
-== Image Caching Configuration
+[[explicit-image-caching]]
+== Explicit Image Caching
 
-To enable image caching you use the xref:../command-ref/minishift_config_set#[`minishift config set`] command:
+{project} provides the `image` command together with its sub-commands to control the image caching behavior.
+To export and import images from the Docker daemon of the {project} VM, you need to run `minishift image export` respectively `minishift image import`.
+
+[[single-images]]
+=== Importing and Exporting Single Images
+
+Once the {project} VM is running, images can be explicitly exported from the Docker daemon by:
 
 ----
-$ minishift config set image-caching true
+$ minishift image export <image-name-0> <image-name-1> ...
+Pulling image <image-name-0> .. OK
+Exporting <image-name-0>. OK
+Pulling image <image-name-1> .. OK
+Exporting <image-name-2>. OK
 ----
 
-Once enabled, caching occurs transparently, in a background process, the first time you use the xref:../command-ref/minishift_start#[`minishift start`] command.
-Once the images are cached under *_$MINISHIFT_HOME/cache/images_*, successive {project} VM creations will use these cached images.
+[NOTE]
+====
+Images which are not available in the Docker daemon will be pulled prior to be exported to the host.
+====
 
-Each time an image exporting background process runs, a log file is generated under *_$MINISHIFT_HOME/logs_* which can be used to verify the progress of the export.
+To import a previously cached images use the xref:../command-ref/minishift_image_import.adoc#[`minishift image import`] command:
 
-You can disable the caching of the OpenShift images by setting `image-caching` to `false` or removing the setting altogether using xref:../command-ref/minishift_config_unset#[`minishift config unset`]:
+----
+$ minishift image import <image-name-0> <image-name-1> ...
+Importing <image-name-0> . OK
+----
+
+Using alpine:3.6 as an example image to exi
```

Comment:
@hferentschik  `.... example image to exi` seems to  be incomplete here.

Reply:
ahh, thanks